app/src/main/java/id/rizmaulana/covid19/data/mapper/CovidDataMapper.kt

```diff
@@ -20,25 +20,24 @@ object CovidDailyDataMapper {
             response.incrementRecovered,
             response.incrementConfirmed
         )
-    }
+    } ?: emptyList()
```

Comment:
Or you can use `orEmpty`

```suggestion
    }.orEmpty()
```

Reply:
Nice info thank you 🙏